### PR TITLE
Docs - awscc_opensearchserverless_collection

### PR DESCRIPTION
**Pull request by troyparrett**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request
* The resources and data sources in this provider are generated from the CloudFormation schema, so they can only support the actions that the underlying schema supports. For this reason submitted bugs should be limited to defects in the generation and runtime code of the provider. Customizing behavior of the resource, or noting a gap in behavior are not valid bugs and should be submitted as enhancements to AWS via the CloudFormation Open Coverage Roadmap.

### CODE REVIEW

File: examples/resources/awscc_opensearchserverless_collection/simple_collection.tf [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+      awscc_opensearchserverless_security_policy.security_policy
+    ]
+}
+
+# Encryption SecurityPolicy
+resource "awscc_opensearchserverless_security_policy" "security_policy" {
+    name = "awscc-security-policy"
+    description = "created via awscc"
+    type = "encryption"
+    policy = "{\"Rules\":[{\"ResourceType\":\"collection\",\"Resource\":[\"collection/awscc-collection\"]}],\"AWSOwnedKey\":true}"
```
Comment (wellsiau-aws) [inline examples/resources/awscc_opensearchserverless_collection/simple_collection.tf:14]:
```suggestion
  policy = jsonencode({
    Rules = [
      {
        ResourceType = "collection",
        Resource     = "collection/awscc-collection"
      }
    ],
    AWSOwnedKey = true
  })
```

Comment (wellsiau-aws) [reply examples/resources/awscc_opensearchserverless_collection/simple_collection.tf:14]: use `jsonencode` to make the policy readable

**Review by wellsiau-aws — CHANGES_REQUESTED** [review]

**Comment by troyparrett** [comment]
@wellsiau-aws - made the changes, good suggestion

File: examples/resources/awscc_opensearchserverless_collection/simple_collection.tf [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+      awscc_opensearchserverless_security_policy.security_policy
+    ]
+}
+
+# Encryption SecurityPolicy
+resource "awscc_opensearchserverless_security_policy" "security_policy" {
+    name = "awscc-security-policy"
+    description = "created via awscc"
+    type = "encryption"
+  policy = jsonencode({
```
Comment (wellsiau-aws) [inline examples/resources/awscc_opensearchserverless_collection/simple_collection.tf:14]: careful with the indent here, its always a good practice to run `terraform fmt` on your local repo

**Review by wellsiau-aws — CHANGES_REQUESTED** [review]
very minor nit-pick about the identation

**Comment by troyparrett** [comment]
done!

**Review by wellsiau-aws — APPROVED** [review]
LGTM, thanks!

**Comment by breathingdust** [comment]
LGTM 🚀 , thanks for the contribution @troyparrett!

**Pull request merged**